.github/workflows/testing_and_validation.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           . dmod_venv/bin/activate
           pip install --upgrade pip
           deactivate 
-          ./scripts/update_package.sh --venv dmod_venv --no-service-packages --dependencies
+          ./scripts/update_package.sh --venv dmod_venv --dependencies
```

Comment:
Yeah, I wasn't sure why that flag was present either. That ended up being the reason why the tests were failing (the service pkgs were not being updated). I'm happy to do either. I figured that if I saw `update_package.sh` I would think that it would install all the packages, not just the `lib` packages, so I just left it like this. Im indifferent.

Reply:
No, as I think more about it, what you've done is probably what _needs_ to be done, and that not already being done was a flaw.  And we can fix the part of the flaw applicable here to the data service by fixing it more generally.